docs/blog/2018-11-01-hacktoberfest-wrapup/index.md

```diff
@@ -5,7 +5,7 @@ author: "Amberley Romo"
 tags: ["hacktoberfest", "open source"]
 ---
 
-Just like that, Gatsby Hacktoberfest is over! Or, "Hacktoverfest"? 😂 (Thanks for the pun, [@m-allanson](https://github.com/m-allanson)). We had a great time Hacktoberfesting with y'all and wanted to share a bit of our planning process and lessons learned in review!
+Just like that, Gatsby Hacktoberfest is over! Or, "Hacktoberfest"? 😂 (Thanks for the pun, [@m-allanson](https://github.com/m-allanson)). We had a great time Hacktoberfesting with y'all and wanted to share a bit of our planning process and lessons learned in review!
```

Comment:
This isn't a typo, but an intended pun 😆

Reply:
I created a PR for it here #15972